packages/grpc-js-core/src/channel.ts

```diff
@@ -214,7 +212,7 @@ export class Http2Channel extends EventEmitter implements Channel {
       address: string, readonly credentials: ChannelCredentials,
       private readonly options: Partial<ChannelOptions>) {
     super();
-    if (credentials.getSecureContext() === null) {
+    if (credentials.isSecure()) {
```

Comment:
Is this boolean check backwards?

Reply:
Yes. I switched the outcomes to avoid a negative if check.